lgtm.yml

```diff
@@ -2,5 +2,3 @@ extraction:
   csharp:
     index:
       buildless: true
-      dotnet:

```

Comment:
This config line is for lgtm to know what analyzers to run... how is it knowing now? I see the check is still passing does that mean this was optional all along or is it reading the lgtm.yml from our repo root branch or what?

Reply:
LGTM reads the global.json file: https://lgtm.com/help/lgtm/csharp-extraction. Seems like this file is now the standard way for builds and other tools to detect the version: https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x.

Reply:
Oh cool that is neat, thanks for pointing me to that